src/app/file-manager/reducers/transferables.reducer.ts

```diff
@@ -99,11 +151,13 @@ export function TransferablesReducer(state = initialState, action: Action): Tran
 
         case TransferablesActions.ADD_ITEM:
             const add_item = [...state.items, action.payload];
+            sortList(add_item);
```

Comment:
I know it will result in sorting the list in a few cases where sorting isn't necessary, but I think it would be better to sort the list in the constructor for `TransferableState`. That avoids the duplicate code here and better documents that the list is always sorted.